### PR TITLE
https://issues.redhat.com/browse/ACM-10623 Update RHEL version

### PR DESCRIPTION
**Pull request by oafischer**
2.10 only

### CODE REVIEW

File: gitops/gitops_manage_policy_def.adoc [hunk 1]
```diff
@@ -115,11 +115,10 @@ spec:
     initContainers:
     - args:
       - -c
-      - cp /etc/kustomize/plugin/policy.open-cluster-management.io/v1/policygenerator/PolicyGenerator
-        /policy-generator/PolicyGenerator
+      - cp /policy-generator/PolicyGenerator-not-fips-compliant /policy-generator/PolicyGenerator
```
Comment (brian-jarvis) [inline gitops/gitops_manage_policy_def.adoc]: The initContainer.volumeMounts.mountPath and cp destination would need to be changed based on details outlined in https://issues.redhat.com/browse/ACM-10623?focusedId=24416968&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-24416968

```
    initContainers:
    - args:
      - -c
      - cp /policy-generator/PolicyGenerator-not-fips-compliant /policy-generator-tmp/PolicyGenerator
      command:
      - /bin/bash
      image: registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel9:v<version>
      name: policy-generator-install
      volumeMounts:
      - mountPath: /policy-generator-tmp
        name: policy-generator
```

**Review by brian-jarvis — CHANGES_REQUESTED** [review]

**Review by swopebe — APPROVED** [review]

File: gitops/gitops_manage_policy_def.adoc (hunk 1 above)
Comment (oafischer) [reply gitops/gitops_manage_policy_def.adoc]: Updated

Comment (brian-jarvis) [reply gitops/gitops_manage_policy_def.adoc]: Note the cp  command also needs to have the -tmp added to the path

cp /policy-generator/PolicyGenerator-not-fips-compliant /policy-generator-tmp/PolicyGenerator

Comment (oafischer) [reply gitops/gitops_manage_policy_def.adoc]: Good catch, updated as well, thanks!

**Review by mprahl — APPROVED** [review]

**Review by brian-jarvis — APPROVED** [review]

**Pull request merged**